usaspending_api/common/helpers/fiscal_year_helpers.py

```diff
@@ -222,6 +222,21 @@ def is_final_period(period: int) -> bool:
     return period == 12
 
 
+def is_valid_monthly_period(year: int, period: int) -> bool:
```

Comment:
Period 1 is also not a valid Fiscal Period in all cases because we group 1 and 2 together.

Reply:
Also thinking this might belong more in a helper under submissions app because all of those are valid fiscal periods, just not for submissions.

Reply:
Moved to submission helpers and removed period 1